modules/acknowledgements/php/module.class.inc

```diff
@@ -24,4 +24,43 @@ namespace LORIS\acknowledgements;
  */
 class Module extends \Module
 {
+    /**
+     * {@inheritDoc}
+     *
+     * @param \User $user The user whose access is being checked.
+     *
+     * @return bool whether access is granted
+     */
+    public function hasAccess(\User $user) : bool
+    {
+        if (parent::hasAccess($user) == false) {
+            return false;
+        }
+        return $user->hasAnyPermission(
+            [
+                'acknowledgements_view',
+                'acknowledgements_edit'
+            ]
+        );
```

Comment:
This may be better written as 

```php
return parent::hasAcess($user)
    && $user->hasAnyPermission(
    [
        'acknowledgements_view',
        'acknowledgements_edit'
    ]
);

Reply:
I somewhat disagree about which one is easier to read, but I don't feel strongly about it so I can change it..

Reply:
It's not a big deal

Reply:
I updated them.